### ISSUE TITLE
Flowchart shapes too small for text in Chromium browsers

### ISSUE THREAD

**Issue by HerbCaudill** [mermaid-js/mermaid#1891]
In Chrome and Edge, flowchart shapes are drawn too small for the text. Unless I'm crazy, this just started happening. Perhaps it's due to a recent Chromium update? 

Firefox renders correctly, as do Safari & Chrome on iOS. (Chrome on iOS uses the same rendering engine as Safari.) 

**To Reproduce**

Visit https://mermaid-js.github.io/mermaid-live-editor in Chrome or Edge version 88. 

**Screenshots** 

Left to right (Windows 10): Firefox 85 👍, Edge 88 👎, Chrome 88 👎 

![image](https://user-images.githubusercontent.com/2136620/107785899-76531d80-6d4d-11eb-86ef-c07a04ab8f2b.png)

Safari (iOS) 👍

<img width=300 src=https://user-images.githubusercontent.com/2136620/107786928-be267480-6d4e-11eb-98ef-dcfd4efc29c1.png />

Chrome (iOS) 👍

<img width=300 src=https://user-images.githubusercontent.com/2136620/107786854-aa7b0e00-6d4e-11eb-8eb0-74c66d2f590c.png />

**Comment by HerbCaudill** [comment]
Update: Installed an older version of Chrome (v86) and it displays correctly, so it is an issue introduced in either v87 or v88. 

![image](https://user-images.githubusercontent.com/2136620/107789202-98e73580-6d51-11eb-8b2e-c1e8d2e12a91.png)

**Comment by wpbrown** [comment]
Seeing this issue on v88  on Linux as well.

**Comment by prtuson** [comment]
I raised a similar issue with mermaid on Android 14 days ago.
Subsequently, I raised the issue with the Chrome developers with ways in which they could test it. They have assured me that it will be fixed in Chromium v89. Indeed I have this on my linux computer and Edge now works properly.
The problem arose with Chromium v87; Chromium is used for both Edge and Android hybrid apps so it caused a problem. Chrome on Linux is now using Chromium v88, which doesn't work.
So we just need to wait. I hope that Chromium 89 will be released soon, Beta was released at the end of January.
